### DISCUSSION TITLE
Next/image - background image attachment fixed

### DISCUSSION THREAD

**Discussion by f-do** [vercel/next.js#20697] [Help]
### Feature discussion
Hi there, great work with the next/image but i have a question about an specific feature and i think that is not possible to solve. 

### Describe the solution you'd like
My goal is to use the next/image for all known reasons, so i want to use the image as background but as attachment fixed. 
For sure i found: https://image-component.nextjs.gallery/background <- here you moved that in the background with `z-index` but if you have a large page and you can scroll it doesn't work. 

So i search a lot around and don't find any solution what is possible to combine lazy loading with `background-attachment: fixed;`.

### Demo
i tried to get an sample running in codesandbox with `tailwindcss 2.0.2` but codesandbox has node 10 and tailwind will need > 12 in there own next demo. 

### Hope
It will be nice if you can help me with that 👍 

Happy new year and stay save

**Comment by ansonlowzf** [comment]
Have you find the solution?

**Comment by Eftychiou** [comment]
Have you guys found a solution ?

**Comment by f-do** [comment]
Hi not really...

**Comment by Eftychiou** [comment]
Hope they get back to us with this feature soon

________________________________

**Comment by Raphoks** [comment]
Have you find the solution?

**Comment by ddrosatosimode** [comment]
My take on that was to use some css and kind of immitate the background-attachment fixed.
`<div class="parent"><div class="bgwrap"><NextImage alt={card.Title} src={${Bgimage}} layout="fill" objectFit="cover" quality={100} /> </div></div>`
And the css part
.bgwrap {
    position: fixed;
    top: 0;
    left: 0;
    width: 100vw;
    height: 100vh;
    z-index: -1;
}

For only problem was that i had to add some white background to the sections that didnt have one.

**Reply by suhitaga** [reply]
It doesn't really work how it should, though. It just skews the image and makes it smaller to fill in the available space
